franklab_mountainsort/ms4_franklab_pyplines.py

```diff
@@ -371,13 +371,14 @@ def add_curation_tags(dataset_dir, output_dir, firing_rate_thresh=0.01,
         peak_snr_thresh=peak_snr_thresh, mv2file='')
 
 
-def recalc_metrics(dataset_dir, output_dir, firings_in='firings_processed.json',
-                   metrics_to_update='metrics_tagged.json',
+def recalc_metrics(dataset_dir, output_dir, firings_in='firings_processed.mda',
+                   metrics_to_update='metrics_processed.json',
                    firing_rate_thresh=0.01, isolation_thresh=0.95,
                    noise_overlap_thresh=0.03,
-                   peak_snr_thresh=1.5, mv2_file='', opts=None):
+                   peak_snr_thresh=1.5, mv2_file='', manual_only=True):
     '''post-merge, should recalculate metrics and update tags (both tags based
```

Comment:
Doc string doesn't match the parameters in the function

Reply:
Updated!